### ISSUE TITLE
SyntaxError: Unexpected token ...

### ISSUE THREAD

**Issue by hiendv** [eslint/eslint#11887]
**Tell us about your environment**

* **ESLint Version:** 6.0.0
* **Node Version:** 6.14.4 (lts/boron)
* **npm Version:** 6.9.0 

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
"eslintConfig": {
    "env": {
      "browser": true,
      "node": true,
      "jest/globals": true
    },
    "extends": [
      "eslint:recommended",
      "plugin:jest/recommended",
      "standard"
    ],
    "plugins": [
      "jest"
    ],
    "rules": {
      "array-bracket-spacing": [
        "error",
        "always"
      ],
      "jest/no-disabled-tests": "warn",
      "jest/no-focused-tests": "error",
      "jest/no-identical-title": "error",
      "jest/prefer-to-have-length": "warn",
      "jest/valid-expect": "error"
    }
  },
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

I was trying to run lint with eslint but none of the commands work because there is a problem with the engine.

```bash
npx eslint .
```

**What did you expect to happen?**
The linting process is okay.

**What actually happened? Please include the actual, raw output from ESLint.**
```
/my/path/node_modules/eslint/lib/cli-engine/cli-engine.js:257
        ...calculateStatsPerFile(messages)
        ^^^

SyntaxError: Unexpected token ...
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/my/path/node_modules/eslint/lib/cli-engine/index.js:3:23)
```

**Are you willing to submit a pull request to fix this bug?** Not sure what's the problem is so no, for now.

**Comment by g-plane** [comment]
Hey @hiendv , thanks for the issue.

You may need to configure `ecmaVersion`: https://eslint.org/docs/user-guide/configuring#specifying-parser-options .

**Comment by aladdin-add** [comment]
seems the error comes from eslint.

you are using node v6, but it's no longer supported by eslint v6 -- it has been EOL. https://github.com/eslint/eslint/blob/master/docs/user-guide/migrating-to-6.0.0.md#drop-node-6

**Comment by hiendv** [comment]
@aladdin-add True. Specifying the parserOptions does not help. Does the migration apply to node v7? Thanks.

**Comment by g-plane** [comment]
No, you should use Node.js 8 or higher.

**Comment by hiendv** [comment]
@g-plane Thanks.

**Comment by Boboandhaifeng** [comment]
I had the same problem.

**Comment by Boboandhaifeng** [comment]
I had the same problem.

**Comment by Boboandhaifeng** [comment]
I had the same problem.

**Comment by Boboandhaifeng** [comment]
I had the same problem.     node Version: 10.15.3

**Comment by kaicataldo** [comment]
@Boboanshaifeng Please make a new issue and fill out the template so we can assist you. Thanks!
